### ISSUE TITLE
Add preventGestureDelay Option to flutter_webview Plugin

### ISSUE THREAD

**Issue by Codewealth** [flutter/flutter#154496]
### Use case

Problem Description: The problem I'm facing is a noticeable delay in touch interactions within the flutter_webview plugin. This delay, which typically occurs between tapping a button and being able to scroll or perform another action, degrades the user experience by making the app feel unresponsive. This issue is particularly problematic in web-based mobile applications where immediate feedback from touch interactions is crucial.

Relation to a Problem: The issue is tied to Flutter’s DelayingGestureRecognizer, which introduces a delay to differentiate between single and double taps. While this may be useful in certain contexts, it hinders the fluidity of interactions in a WebView, where immediate touch response is often more important. This is especially frustrating in applications that require quick, responsive interactions, like scrolling immediately after tapping a button.

Alternative Solutions Considered: I have considered using alternative packages like flutter_inappwebview, which offers a preventGestureDelay option that effectively eliminates this delay. However, this package hasn't been maintained for several months, leading to concerns about long-term support and compatibility. Additionally, the flutter_webview plugin is the official and more stable option, making it the preferred choice for many developers, including myself.

### Proposal

Feature Request: I propose adding a preventGestureDelay option to the flutter_webview plugin. This option should allow developers to toggle the prevention of gesture delays, specifically addressing the delay caused by the DelayingGestureRecognizer. This will enable WebView to respond to touch interactions, such as taps and scrolls, immediately, improving the overall user experience.

Why This Should Be Provided by Flutter: Given that flutter_webview is the official WebView plugin for Flutter, it is crucial for it to support common use cases that demand responsive touch interactions. This feature would enhance the plugin's utility and ensure it meets the needs of modern mobile applications that require quick, fluid touch gestures. While this could potentially be implemented in a third-party package, having it as part of the official plugin ensures better integration, maintenance, and stability across Flutter projects.

**Comment by darshankawar** [comment]
@Codewealth 
Can you take a look at this issue with underlying comments to see if they resonate with your case ?

**Comment by Codewealth** [comment]
> @Codewealth Can you take a look at [this](https://github.com/flutter/flutter/issues/94325) issue with underlying comments to see if they resonate with your case ?

Yes, that's exactly the issue I'm describing. I found a workaround for our project but it would be good to have it as an option in the next release.

**Comment by darshankawar** [comment]
Thanks for the update. I will keep this issue open as proposal.

Related issue: https://github.com/flutter/flutter/issues/94325
